dataless-classifier/README.md

```diff
@@ -0,0 +1,59 @@
+# Illinois-DatalessClassification
```

Comment:
CogComp instead of Illinois

Reply:
Can you add an introduction paragraph here? Like for someone who has no idea about the relevant papers, in simple words, what is the input and the corresponding output? You may put some example input / outputs too. 
Some of the content in the "Label Hierarchy" might be useful for here too.